### DISCUSSION TITLE
Can this happen ? If this can so just give me the example

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#138] [Q&A]
The idea is that when we press the button it will scroll to y: 2000 . Can this happen?

If this can so give me a little example code. Docs don't really provide that much info about it

**Comment by MyoungXUE** [comment]
Hello, use the hmApp.setLayerY API in the button's click event https://docs.zepp.com/docs/1.0/reference/device-app-api/hmApp/setLayerY/
